apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +19,33 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+
+  it('Then: I should see mark book as finished', async () => {
```

Comment:
Your test fails:
```
1) When: I use the reading list feature Then: I should see mark book as finished
  Message:
    Failed: element not interactable
```

(it cannot find the element you want it to click)

Reply:
My bad, missed to include open reading list before marking book as read. Updated e2e spec. Demo of e2e test - https://recordit.co/PPy9C2GTIc